core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -432,26 +432,23 @@ private void checkTempBlockOwnedBySession(long sessionId, long blockId)
    */
   private void abortBlockInternal(long sessionId, long blockId) throws BlockDoesNotExistException,
```

Comment:
Update the javadoc of this class to mention abortBlock does not acquire block lock. It mentions createBlock.